protocol/js/flow-types-chat.js

```diff
@@ -16,6 +16,60 @@ export type RPCError = {
   code: number,
   desc: string
 }
+export type WaitingHandlerType = (waiting: boolean, method: string, sessionID: number) => void
+
+// $FlowIssue we're calling an internal method on engine that's there just for us
+const engineRpcOutgoing = (...args) => engine._rpcOutgoing(...args)
```

Comment:
so this code already existed. (not that we can't change it). this is so people who are making rpc calls don't need to understand that engine exists else they'll all have to import it just to pass it into this helper (which seems lame).
new engines are not being made, its a singleton thats exported by the module. if we want to export a closure that also exports a singleton thats fine but that isn't a fix for what you're talking about